tests/utilities.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var assert = require('assert');
+var rewire = require('rewire');
```

Comment:
Too bad that we need this wizardry to write our unit tests. But if they're OK with it upstream, I won't complain.